docs-ref-services/preview/resourcemanager-databox-readme.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Resource Manager DataBox client library for Java
+description: "Azure Resource Manager DataBox client library for Java - Version 1.0.0-beta.1 contains Microsoft Azure SDK for DataBox Management SDK.  Package tag package-2021-03." 
```

Comment:
You have an extra space before the 3rd sentence, which I think can be removed from the description field.  I don' think "Package tag..." helps in any search results.

Reply:
fixed.